layouts/catalog/list.html

```diff
@@ -86,6 +86,14 @@ <h1 id="works">
               {{ end }}
             </ul>
             {{ end }}
+            {{ with .additional_resources }}
+              <span class="b mv0">Additional Resources</span>
+              <ul>
+              {{ range . }}
+                <li><a href="#{{ .url }}">{{ .label }}</a>{{ if .description }}: {{ .description }} {{ end }}</li>
```

Comment:
We should leave off the `#` since these are external links.

```suggestion
                <li><a href="{{ .url }}">{{ .label }}</a>{{ if .description }}: {{ .description }} {{ end }}</li>
```